### PR TITLE
Add online compiler steps to Migrating documentation

### PR DESCRIPTION
**Pull request by yennster**
- Added steps to upgrade to Mbed OS 5 from Mbed OS 2 or RTOS from within the online compiler
- Added screenshots for the steps into the images folder

### CODE REVIEW

**Comment by AnotherButler** [comment]
@yennster Thanks so much 👍 

@MarceloSalazar @ashok-rao As @janjongboom raised this issue but is out for a few weeks, could one of y'all please review these changes on his behalf?

@jen3andruska Once the content has been approved, could you please do a quick copy edit of these changes and add captions to the images? [Note: Any release work takes priority over this.]

**Comment by jen3andruska** [comment]
@AnotherButler Sounds great.  I'll wait for the content to be approved, and then do a copy edit and add captions to the images.

File: docs/tutorials/migrate-from-mbed-os-2-to-5.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ To determine the success of migration, run:
 
 `mbed compile -m [platform] -t [toolchain]`
 
+The general outline for updating to Mbed OS 5 with the [Mbed Online Compiler](https://os.mbed.com/docs/latest/tools/arm-mbed-online-compiler.html) is:
+
+1. Open your project in the online compiler
+1. Right click on `mbed` and select "Delete..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-delete-mbed.png)</span>
+1. Right click on `mbed-rtos` and select "Delete..."
```
Comment (ashok-rao) [inline docs/tutorials/migrate-from-mbed-os-2-to-5.md]: @yennster : I didn't find the "mbed-rtos" directory in blinky example for Mbed classic when I import into online compiler from here: https://os.mbed.com/teams/mbed/code/mbed_blinky/  .. you may want to get rid of this line?? or am I missing something?

Comment (yennster) [reply docs/tutorials/migrate-from-mbed-os-2-to-5.md]: @ashok-rao You're right, Mbed OS 2 blinky doesn't use Mbed RTOS (aka Mbed OS 3). I'll add a clarifying statement saying something like "If your project also uses `mbed-rtos` you'll need to delete this library as well in order to successfully upgrade to Mbed OS 5..."

**Comment by yennster** [comment]
@ashok-rao done 👍

File: docs/tutorials/migrate-from-mbed-os-2-to-5.md [hunk 2]
```diff
@@ -48,6 +48,20 @@ To determine the success of migration, run:
...
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-delete-mbed.png)</span>
+1. If your project includes `mbed-rtos`, you will also need to delete this library to successfully update to Mbed OS 5. Right click on `mbed-rtos` and select "Delete..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-delete-mbed-rtos.png)</span>
+1. Right click on the name of your project and hover over "Import Library" then select "From URL ..."
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/mbed2-import-url.png)</span>
+1. Copy and paste the URL for Mbed OS 5 `https://github.com/armmbed/mbed-os` then click "Import"
+   <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/import-mbed-os.png)</span>
+
+To determine the success of migration, select your board in the top right corner of the Online Compiler, then click  **Compile**.
+
```
Comment (jen3andruska) [inline docs/tutorials/migrate-from-mbed-os-2-to-5.md]: Can't comment on line #142 below, but we don't use '/', so I'm assuming that you mean "replace the SDA [and] SCL pins with the ones present on the K64" and that you mean "D14 [and] D15 work"??  Let me know if the '/' means 'and' in these two places.

Comment (jen3andruska) [reply docs/tutorials/migrate-from-mbed-os-2-to-5.md]: Also, in line #290 below, after "Line 114 of the `AD7790.h` header file now looks like this:" I think there's an extra space after the first '(' below in line #290 - the others don't have this - is this an error?

**Comment by jen3andruska** [comment]
@AnotherButler @yennster I've finished copy-editing and added image captions.  I left a few questions above about line #142 and line #290.  Let me know, and otherwise should be good to go!

Also, the images under steps 2, 3 & 4 of 'Update to Mbed OS 5 with the Mbed Online Compiler' are not appearing.  Mike Q said they have not yet been uploaded, but he can do that when they are merged to the development branch - so, let him know once you merge.

File: docs/tutorials/migrate-from-mbed-os-2-to-5.md (hunk 2 above)
Comment (yennster) [reply docs/tutorials/migrate-from-mbed-os-2-to-5.md]: @jen3andruska Sorry, I'm confused about your comments. Which line numbers are you referring to? Those links go to PR's

**Comment by yennster** [comment]
@jen3andruska I put the images in the same format as how they will appear when they are uploaded onto the AWS server, are my images in the correct format @AnotherButler ?

**Comment by jen3andruska** [comment]
@yennster I've checked the images and they should be in the correct format - this is the same format as the other images.  I was just confused because some of the images are displaying correctly right now, while these three do not (perhaps these are the only three not yet uploaded?).  @AnotherButler ?

**Comment by AnotherButler** [comment]
@jen3andruska Yes, we'll ask for the new images to be uploaded once we merge the PR to the test site.

**Comment by jen3andruska** [comment]
@AnotherButler Sounds great!  Yes, I think its ready to merge.

File: docs/tutorials/migrate-from-mbed-os-2-to-5.md (hunk 2 above)
Comment (jen3andruska) [reply docs/tutorials/migrate-from-mbed-os-2-to-5.md]: @yennster Ok, its now line #300 - it says: 
AD7790( float reference_voltage, PinName CS, PinName MOSI, PinName MISO, PinName SCK);
Is there supposed to be a space after the '('   after AD7790?
The other header files do not have a space after the '(' and thought it should probably be deleted

Comment (yennster) [reply docs/tutorials/migrate-from-mbed-os-2-to-5.md]: @jen3andruska Oh yeah, that space is a typo. Shouldn't cause any compiling issues, however

Comment (yennster) [reply docs/tutorials/migrate-from-mbed-os-2-to-5.md]: @jen3andruska Fixed

**Pull request merged**